submissions/TatyanaMolchanova/js-dom/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Site with menu</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <div class="content">
+
+        <aside class="aside">
+            <ul class="menu">
+                <li class="menu__item active" data-item="animal1">Cats</li>
+                <li class="menu__item" data-item="animal2">Dogs</li>
+                <li class="menu__item" data-item="animal3">Lions</li>
```

Comment:
Try to create a menu with JS using methods `innerHTML, insertAdjacentHTML, append, createElement, addEventListener ......`

Reply:
I create a menu with JS methods: createElement, innerHTML, append.